### ISSUE TITLE
bootkube.service fails due to missing it's WorkingDirectory, and assets.b64

### ISSUE THREAD

**Issue by zendeavor** [coreos/tectonic-installer#846]
## Versions
Tectonic: 1.6.2
Terraform: ?
Platform: metal

## What happened?
After working around #840, I am immediately met with a failure in `bootkube.service` that may or may not be identical or related to #797, which needs to be clarified.

In my case, there is no `/opt/bootkube` directory. Then there is no `assets.b64` to decode, which results in no `assets.zip` to decompress.

## What you expected to happen?
`/opt/bootkube` should exist. `assets.b64` should exist at that directory.

## How to reproduce it (as minimally and precisely as possible)?
I ran into this at the final submission step of the tectonic installer. As far as I can tell, these files just fail to make the trip from the installer machine to the controller/master, so attempting a bare metal installation should result in this same error.

**Comment by s-urbaniak** [comment]
I unfortunately cannot reproduce this issue.

The working directory `/opt/bootkube` is not being used. `bootkube.service` uses `WorkingDirectory=/opt/tectonic`. Also `assets.zip` is not being used in the baremetal case. Here assets are being `scp`'d to the first master machine under `/opt/tectonic`.

Your initial issue report from https://github.com/coreos/tectonic-installer/issues/840#issue-231037355 rather indicates that there were rather issues provisioning the ignition templates to your masters, where `bootkube.service` (https://github.com/coreos/tectonic-installer/blob/master/platforms/metal/cl/bootkube-controller.yaml.tmpl#L85) and `tectonic.service` (https://github.com/coreos/tectonic-installer/blob/master/platforms/metal/cl/bootkube-controller.yaml.tmpl#L101) is being rendered, way before the above mentioned `scp` kicks in.

So, I believe to triage the issue, I would try to check if the mac addresses of the master nodes match with those you specified in the installer.

`systemctl cat bootkube.service` on the target master machine must output something like this:

```
core@master1 /opt/tectonic $ systemctl cat bootkube.service
# /etc/systemd/system/bootkube.service
[Unit]
Description=Bootstrap a Kubernetes cluster
ConditionPathExists=!/opt/tectonic/init_bootkube.done
[Service]
Type=oneshot
RemainAfterExit=true
WorkingDirectory=/opt/tectonic
User=root
Group=root
ExecStart=/usr/bin/bash /opt/tectonic/bootkube.sh
ExecStartPost=/bin/touch /opt/tectonic/init_bootkube.done
[Install]
WantedBy=multi-user.target
```

before it can be started. I.e. the following mac address:

```
core@master1 /opt/tectonic $ ip link show dev ens3 
2: ens3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP mode DEFAULT group default qlen 1000
    link/ether 0a:0b:0c:0d:0e:10 brd ff:ff:ff:ff:ff:ff
```

in my case being `0a:0b:0c:0d:0e:10` must match the configured one in the installer.

**Comment by RaphSfeir** [comment]
I'm having the same issue as the initial one ([#840](https://github.com/coreos/tectonic-installer/issues/840#issue-231037355)).

I'm running tectonic installer 1.6.2 and tried installing on bare metal nodes : one provisioner, one worker and one controller node as required by the instructions. All nodes run Container Linux by CoreOS (1353.8.0 stable). They're running a RPN group on Online.net.

Basically, all I did was install CoreOS on all nodes, configure the provision node with `matchbox` and `dnsmasq` following the guide, configure my cloud config for `ssh`, `etcd` and `fleet` on all nodes. Verifying the `matchbox` endpoints and running `etcdctl cluster-health` works.

But running the installer I get the error: `Failed to start tectonic.service: Unit tectonic.service not found.`

I checked the private mac address configured in the installer : 
```
pow@pow-core-1 ~ $ ip link show enp0s20f1
3: enp0s20f1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc mq state UP mode DEFAULT group default qlen 1000
    link/ether 00:07:cb:04:00:2f brd ff:ff:ff:ff:ff:ff
```

Note that as it is RPN, there's a public and private address. I've tried both mac adresses on my installer multiple times but still get the error. Any chance I'm missing something in my provisioning configuration?
